README.md

```diff
@@ -1 +1,206 @@
-# leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./assets/pf-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h1><b>Hello visitor, welcome to my Project</b></h1>
+
+</div>
+
+
+<div align="center">
+  
+  <br/>
+<h1 align="center">Leaderboard</h1>
+</div>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+
+**To-Do List** is a project to send and get data from API.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/TR/2011/WD-html5-20110405/">HTML5</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/specs.en.html">CSS</a></li>
+    <li><a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-262/">JavaScript</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Use ES6 modules and webpack**
+- **Create it dynamically** 
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://otmaneechchafyky.github.io/leaderboard/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Clone this repo. 
+- Change whatever you like.
+
+
+### Prerequisites
+
+To run this project you need: To clone or fork and run it in a browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/otmaneechchafyky/leaderboard.git
+```
+
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+### Install
+
+- n/a
+
+### Usage
+
+To run the project, open with **index.html** with any web browser.
+
+For example using Linux:
+
+```sh
+  cd leaderboard
+  google-chrome index.html
+```
+
+### Run tests
+
+- n/a
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] To make your documentation be more professional and descriptive to non technical users. I think it would be better if you add all needed information by users which can help them of how to `access` the cloned directory, `install` dependencies, `run build` for production, `all tools` needed, `run your app` locally,... I recommend you use this **[template as an example](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** to fill your `Getting Started` section and remove some sub-section that don't have related information to your project(if exist).

Reply:
done